### PR TITLE
K8s on VMs

### PR DESCRIPTION
**Pull request by Shillaker**
- Multi-VM provisioning
- Split docs into Azure-specific, general k8s, general VMs
- Moved all non-Azure-specific tasks related to `k8s` into separate `tasks.k8s.py`
- Added Ansible scripts for installing k8s on set of VMs, and for basic VM configuration

### CODE REVIEW

File: ansible/inventory/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Inventory files in this directory need to be of the form:
+
+```
+[all]
+52.170.26.16
+13.82.111.202
+
+[main]
```
Comment (csegarragonz) [inline ansible/inventory/README.md:8]: Is `main` an ansible term?

File: ansible/tasks/base.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+- name: "System deps"
+  become: yes
+  apt:
+    name:
+      - apt-transport-https
+      - ca-certificates
+      - curl
+      - gnupg2
+      - pass
```
Comment (csegarragonz) [inline ansible/tasks/base.yml:11]: I thought `pass` was a CLI to manage passwords using `gnupg2`. I am curious, how come we need it?

File: tasks/vm.py [hunk 3]
```diff
@@ -257,3 +282,89 @@ def ip(ctx, name):
...
+@task
+def setup(ctx):
+    """
+    Set up an individual VM with the basics
+    """
+    run_ansible_playbook("vm.yml")
+
+
+@task
+def ports(ctx, prefix=None):
```
Comment (csegarragonz) [inline tasks/vm.py:296]: Could we rename to `open_ports`?

**Review by csegarragonz — APPROVED** [review]
Minor comments, but LGTM

File: ansible/tasks/base.yml (hunk 2 above)
Comment (Shillaker) [reply ansible/tasks/base.yml:11]: Yes good question, this list was taken from some prerequisites for Ansible which sometimes does some funky stuff with passwords. Admittedly we may get away without it as we don't really use any of those kind of features.

**Pull request merged**